jax/_src/scipy/cluster/vq.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import operator
+
+import scipy.cluster.vq
+import textwrap
+
+from jax import vmap
+from jax._src.numpy.util import _wraps, _promote_args_inexact
+from jax._src.numpy.lax_numpy import argmin
+from jax._src.numpy.linalg import norm
+
+
+_no_chkfinite_doc = textwrap.dedent("""
+Does not currently support the Scipy argument ``check_finite=True``,
+because `jax.numpy.asarray_chkfinite` does not exist at the moment.

```

Comment:
I think this is more funcamental: rather than "doesn't exist at the moment", perhaps say "because compiled JAX code cannot perform checks of array values at runtime".

Reply:
Agreed. - I just followed the existed docstring for `scipy.linalg.rsf2csf`. Do you think I should change all check finite docstring?

Reply:
All things being equal, my preference would be that a PR titled "implement scipy.cluster.vq.vq" not touch a bunch of unrelated files.

Reply:
Sorry to be dense, I would split these docstring change into another PR.